### PR TITLE
[Issue #605] remove guard daemon and revise scripts

### PR DESCRIPTION
**Pull request by bianhq**
Remove the guard daemon in Pixels to simplify the start process and maintenance.
Revise the scripts accordingly.

### CODE REVIEW

File: scripts/sbin/start-datanode.sh [hunk 1]
```diff
@@ -1,19 +1,20 @@
...
 
-while read datanode home
+while read -r datanode home
 do
     home="${home:-${DEFAULT_PIXELS_HOME}}"
-    REMOTE_SCRIPT="export PIXELS_HOME=${home} && $PIXELS_HOME/bin/start-datanode.sh -daemon"
-    echo "Starting DataNode on ${datanode}."
+    REMOTE_SCRIPT="export PIXELS_HOME=${home} && $PIXELS_HOME/bin/start-daemon.sh datanode -daemon"
+    echo "Starting datanode on ${datanode}..."
     ssh "${datanode}" "${REMOTE_SCRIPT}"
```
Comment (kkzzjx) [inline scripts/sbin/start-datanode.sh:16]: Use `ssh -n "${datanode}" "${REMOTE_SCRIPT}"` instead.

File: scripts/sbin/stop-datanode.sh [hunk 2]
```diff
@@ -1,16 +1,17 @@
...
 
-while read datanode home
+while read -r datanode home
 do
     home="${home:-${DEFAULT_PIXELS_HOME}}"
-    REMOTE_SCRIPT="export PIXELS_HOME=${home} && $PIXELS_HOME/bin/stop-datanode.sh"
-    echo "Stop DataNode on ${datanode}."
+    REMOTE_SCRIPT="export PIXELS_HOME=${home} && $PIXELS_HOME/bin/stop-daemon.sh datanode"
+    echo "Stop datanode on ${datanode}."
     ssh "${datanode}" "${REMOTE_SCRIPT}"
```
Comment (kkzzjx) [inline scripts/sbin/stop-datanode.sh:16]: Use `ssh -n "${datanode}" "${REMOTE_SCRIPT}"` instead.

Comment (bianhq) [reply scripts/sbin/stop-datanode.sh]: Fixed.

File: scripts/sbin/start-datanode.sh (hunk 1 above)
Comment (bianhq) [reply scripts/sbin/start-datanode.sh]: Fixed.

**Review by kkzzjx — APPROVED** [review]

**Review by kkzzjx — APPROVED** [review]

**Pull request merged**